build error for yaz 5.34.0 with glibc 2.38.9000

With glibc 2.39 development version, yaz 5.34.0 fails to build as:

```
xmlquery.c:56:23: error: implicit declaration of function 'atoi'; did you mean 'atoi_n'? [-Werror=implicit-function-declaration]
record_conv.c:814:21: error: implicit declaration of function 'atoi'; did you mean 'atoi_n'? [-Werror=implicit-function-declaration]
```

Also:

```
test_record_conv.c:41:34: error: implicit declaration of function 'getenv' [-Werror=implicit-function-declaration]
test_retrieval.c:36:34: error: implicit declaration of function 'getenv' [-Werror=implicit-function-declaration]
```

--- a/src/record_conv.c
+++ b/src/record_conv.c
@@ -11,6 +11,7 @@
 #include <config.h>
 #endif
 
+#include <stdlib.h>
 #include <string.h>
 #include <yaz/log.h>
 #include <yaz/yaz-iconv.h>

--- a/src/xmlquery.c
+++ b/src/xmlquery.c
@@ -10,6 +10,7 @@
 #endif
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include <assert.h>
 

--- a/test/test_record_conv.c
+++ b/test/test_record_conv.c
@@ -9,6 +9,7 @@
 #include <yaz/record_conv.h>
 #include <yaz/test.h>
 #include <yaz/wrbuf.h>
+#include <stdlib.h>
 #include <string.h>
 #include <yaz/log.h>
 #include <yaz/proto.h>

--- a/test/test_retrieval.c
+++ b/test/test_retrieval.c
@@ -9,6 +9,7 @@
 #include <yaz/retrieval.h>
 #include <yaz/test.h>
 #include <yaz/wrbuf.h>
+#include <stdlib.h>
 #include <string.h>
 #include <yaz/log.h>
 #include <yaz/oid_db.h>

